docs/en/latest/concepts/annotations.md

```diff
@@ -204,3 +204,56 @@ spec:
             port:
               number: 80
 ```
+
+Use ApisixPluginConfig
+---------
+
+You can use the following annotations to use the `ApisixPluginConfig`.
+
+* `k8s.apisix.apache.org/plugin-conifg-name`
```

Comment:
there is a typo in annotation name:
`k8s.apisix.apache.org/plugin-config-name`

Reply:
@pietrogoddibit2win This page was updated after merging this PR. I think the issue is now resolved: https://github.com/apache/apisix-ingress-controller/blob/master/docs/en/latest/concepts/annotations.md

Reply:
In the official docs page I can still see the error https://apisix.apache.org/docs/ingress-controller/concepts/annotations/

Reply:
Ah, that's because these fixes have not been published yet. We are trying to make the versioning better, but now it is a mess. You can select "Next" from the version dropdown and you would be able to see the updated docs.